src/software/simulation/ssl_simulation_protocol/ssl_simulation_protocol_network_module.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+class SslSimulationProtocolNetworkModule {
```

Comment:
Shouldn't this receive primitives and vision msgs as well?

Reply:
Sure, that'd be fine too - as is, I made this module only responsible for communication with the simulator

Reply:
That's fine too, making two different modules